sdk/search/azure-search-documents/src/main/java/com/azure/search/documents/SearchClient.java

```diff
@@ -229,15 +229,6 @@ public IndexDocumentsResult deleteDocuments(Iterable<?> documents) {
         return asyncClient.deleteDocumentsWithResponse(documents, context).block();
     }
 
-    /**
```

Comment:
The sync client APIs are missing @ServiceMethod annotation and codesnippets too.